llvmlite/tests/test_ir.py

```diff
@@ -2451,9 +2451,12 @@ def assert_ne(ptr1, ptr2):
     def test_ptr_intrinsic_name(self):
         self.assertEqual(ir.PointerType().intrinsic_name, 'p0')
         self.assertEqual(ir.PointerType(addrspace=1).intrinsic_name, 'p1')
-        # Note: Should this be adjusted based on the pointer mode?
-        self.assertEqual(ir.PointerType(int1).intrinsic_name, 'p0i1')
-        self.assertEqual(ir.PointerType(int1, 1).intrinsic_name, 'p1i1')
+        if not ir_layer_typed_pointers_enabled:
```

Comment:
I think the changes here pertain to the fix I asked about above. Is that right?

Reply:
Exactly, I wasn't sure if we should remove intrinsic names from typed pointers entirely (such that we would always print them in opaque pointer format), but then I thought we should probably keep consistency with `_to_string`, and so I've adjusted this accordingly.